src/widgets/chart.rs

```diff
@@ -201,6 +308,8 @@ pub struct Chart<'a> {
     style: Style,
     /// Constraints used to determine whether the legend should be shown or not
     hidden_legend_constraints: (Constraint, Constraint),
+    /// The position of a legend
+    legend_position: Option<LegendPosition>,
```

Comment:
[b6e6c0d](https://github.com/ratatui-org/ratatui/pull/378/commits/b6e6c0d30f78a129340695026885596a914b1803) add description about `None` position into `legend_position` function. Because `legend_position` is private property.

Reply:
Works for me :+1: